src/main/java/com/bobocode/bibernate/session/SessionImpl.java

```diff
@@ -67,7 +77,7 @@ public <T> Optional<T> find(Class<T> type, Object primaryKey) {
             throw new BibernateException("More than 1 result were found!");
         }
         Optional<T> entity = Optional.of(foundEntities.get(0));
```

Comment:
What the purpose to wrap this in Optional?  if we have already checked that list is not empty and it contains just 1 element? I guess this value can`t be null.